.codeclimate.yml

```diff
@@ -8,5 +8,5 @@ plugins:
 exclude_patterns:
   - config/**/*.js
   - dist/**/*.js
-  - spec/**/*
+  - src/patterns/**/**/*.js"
```

Comment:
Don't we still want codeclimate to run on the actual JS files?

For example: `src/patterns/components/usa-accordion/accordion.js`

Reply:
good catch, yeah should probably be `src/patterns/**/*spec.js`